docs/config.rst

```diff
@@ -423,6 +423,7 @@ storageSubPath      The path in the persistent volume to be mounted (default: ro
 computeResourceType Define whether use Kubernetes ``Pod`` or ``Job`` resource type to carry out Nextflow tasks (default: ``Pod``).
 fetchNodeName       If you trace the hostname, activate this option (default: ``false``, requires version ``22.05.0-edge`` or later).
 volumeClaims        (deprecated)
+maxRequestRetries   Defines the Kubernetes API max request retries (default is set to 5)
```

Comment:
Can we name this `maxErrorRetry` as well so that it's consistent with the analogous options for HPC and Azure?